nncf/common/utils/tensorboard.py

```diff
@@ -65,3 +66,14 @@ def _(stats, algorithm_name):
         tensorboard_stats[f'{algorithm_name}/target_sparsity_level'] = target_sparsity_level
 
     return tensorboard_stats
+
+
+@convert_to_dict.register(MovementSparsityStatistics)
```

Comment:
unit test

Reply:
@yujiepan-work does this https://github.com/vuiseng9/nncf/commit/6ced2896e7d9b836f609add9449fdaf5e60e622d also test this functionality?

Reply:
not needed for now.